### ISSUE TITLE
Bug: Cannot read property 'children' with rendered children array.

### ISSUE THREAD

**Issue by or2008** [FormidableLabs/radium#950]
I am using the new React feature - portals, to render a div outside of the app root element, it works great, however, when wrapping the component with Radium it throws this error:

Uncaught TypeError: Cannot read property 'children' of undefined
    at resolveStyles (resolve-styles.js:317)

![image](https://user-images.githubusercontent.com/3414979/33881550-c47ba51e-df3d-11e7-95e7-dee0dac3091a.png)

**Comment by ryan-roemer** [comment]
@or2008 -- Can you provide a minimal repository with install + error reproduction steps so that we can more efficiently dive in and diagnose? Thanks!

**Comment by alex00x** [comment]
Hi @ryan-roemer  The error is reproduced in case the render method returns an array

Example:
```
class Spinner extends React.Component {
  render() {
    return [
      <Style
        rules={{
          '.smthng': {
            color: 'green'
          }
        }}
      />,
      <div>
        <div
          style={[spinnerStyles.inner, {'@media print': {height: '10px'}}]}
        />
      </div>
    ];
  }
}

Spinner = Radium(Spinner);
```

**Comment by ryan-roemer** [comment]
Verified. WIP branch up at https://github.com/FormidableLabs/radium/compare/bug/render-array with just failing regression test for a start.

@alex00x -- Can you review and confirm that failing test captures the bug?

**Comment by alex00x** [comment]
@ryan-roemer Yep, looks like

**Comment by kylecesmat** [comment]
This bug was fixed & released in `Radium@0.24.0`. If you are still having issues, please re-open but I will close this as resolved :)
